README.md

```diff
@@ -87,7 +87,7 @@ Install this project with:
 To run the project, execute the following command:
 
 <ul>
-<li>ruby mylist.rb</li>
+<li>ruby main.rb</li>
 </ul>
```

Comment:
- [ ] [optional] @torobucci and @peterdtitan it would be better if you added how to run tests in your project.